marketplace-ui/src/app/shared/components/header/header.component.ts

```diff
@@ -25,6 +32,8 @@ import { ThemeSelectionComponent } from './theme-selection/theme-selection.compo
   styleUrls: ['./header.component.scss', '../../../app.component.scss']
 })
 export class HeaderComponent {
+  @Output() isHeaderMenuShown = new EventEmitter<boolean>();
```

Comment:
please remove this one and try to make isMobileMenuCollapsed also as an output of component